Makefile

```diff
@@ -87,6 +87,8 @@ manifests: controller-gen ## Generate WebhookConfiguration, ClusterRole and Cust
 
 generate: controller-gen ## Generate code containing DeepCopy, DeepCopyInto, and DeepCopyObject method implementations.
 	$(CONTROLLER_GEN) object:headerFile="hack/boilerplate.go.txt" paths="./..."
+	GOFLAGS="" go install fybrik.io/crdoc@latest
```

Comment:
Maybe we should do something like:
```
test -f $(go env GOPATH)/bin/crdoc || GOFLAGS="" go install fybrik.io/crdoc@latest
```

Reply:
I guess `go install` already does this checks and also manage updates if available since I pointed @latest